### ISSUE TITLE
Print the filepaths of checked files when the verbose option is activated

### ISSUE THREAD

**Issue by Kristinita** [pylint-dev/pylint#9357]
### 1. Summary

It would be nice if Pylint would print the filepaths of the files it checked to the console. This can be implemented, for example, by adding:

1. The desired behavior if the user uses a CLI argument `--verbose`
1. A new CLI argument, with a name like `--print-filepaths`

If requested behavior is already possible, please add to the documentation what needs to be done to get it.

### 2. Example of desired behavior

The same as implemented, for example, in pydocstyle:

```sh
# [INFO] No pydocstyle output if all docstrings in all files are valid
pydocstyle .

# [INFO] Print filepaths all checked files if “--verbose” argument enabled
pydocstyle . --verbose

Checking file .\pelicanconf.py.
Checking file .\pelican_settings_loader.py.
Checking file .\publishconf.py.
Checking file .\jinja_filters\jinja_filter_all.py.
Checking file .\jinja_filters\jinja_filter_any.py.
Checking file .\personal-plugins\autotitle\autotitle.py.
Checking file .\personal-plugins\autotitle\__init__.py.
Checking file .\personal-plugins\putkiry\putkiry.py.
Checking file .\personal-plugins\putkiry\__init__.py.
```

### 3. Reasons why the feature is needed

1. Checking whether the user has configured Pylint options `ignore`, `ignore-paths` and `ignore-patterns` correctly.
1. Checking that Pylint actually checks all files necessary for the user. For example, earlier [**Pylint don’t supported analyzing directories with no \_\_init\_\_ files in them**](https://github.com/pylint-dev/pylint/issues/2108#issuecomment-389658050). Pylint may not analyze some directories/files, but the user may not understand this.

### 4. Not helped

I couldn’t find how I can get the desired behavior in:

1. [**“Pylint output” page**](https://pylint.pycqa.org/en/latest/user_guide/usage/output.html) of Pylint documentation
1. [**`pylintrc` example on this repository**](https://github.com/pylint-dev/pylint/blob/main/pylintrc)
1. Google. I found a 2017 answer on Stack Overflow, “[**This currently isn’t possible**](https://stackoverflow.com/a/43084405/5951529)” to a similar question.

Thanks.

**Comment by jzohrab** [comment]
Included in the rought draft PR for issue #1554 .

**Comment by jzohrab** [comment]
This is handled in the recently closed PR #10178 .  I think this can be closed.
